server/CONTRIBUTING.md

```diff
@@ -112,10 +111,251 @@ pytest --hge-urls http://127.0.0.1:8080 --pg-urls <database_url> -vv
 - Create a pull request from your forked repo to the main repo.
 - Every pull request will automatically build and run the tests.
 
+
 ## Code conventions
 
 This helps enforce a uniform style for all committers.
 
 - Compiler warnings are turned on, make sure your code has no warnings.
-- Use [hlint](https://github.com/ndmitchell/hlint) to make sure your code has no warnings.
-- Use [stylish-haskell](https://github.com/jaspervdj/stylish-haskell) to format your code.
+- Use [hlint](https://github.com/ndmitchell/hlint) to make sure your code has no
+  warnings.
+- Use [stylish-haskell](https://github.com/jaspervdj/stylish-haskell) to format
+  your code.
+
+This is a short document describing the preferred coding style for this project.
+We've tried to cover the major areas of formatting and naming. When something
+isn't covered by this guide you should stay consistent with the code in the
+other modules.
+
+### Formatting
+
+#### Line Length
+
+Maximum line length is *80 characters*.
```

Comment:
I personally prefer 100 - 120 chars, but don't feel strongly about this.

Reply:
I agree—80 characters seems very small to me, and it seems more likely to encourage artificially short names.

Reply:
Agreed. I'll make this ~100 chars?